v2/thirdpartyemailpassword/serverless/with-aws-lambda/backend-config.mdx

```diff
@@ -57,28 +112,27 @@ function getBackendConfig() {
         providers: [
           // We have provided you with development keys which you can use for testing.
           // IMPORTANT: Please replace them with your own OAuth keys for production use.
-          ThirdPartyEmailPassword.Google({
-            clientId: "1060725074195-kmeum4crr01uirfl2op9kd5acmi9jutn.apps.googleusercontent.com",
-            clientSecret: "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"
+          ThirdParty.Google({
+            clientId: "^{googleClientId}",
+            clientSecret: "^{googleClientSecret}",
           }),
-          ThirdPartyEmailPassword.Github({
-            clientId: "467101b197249757c71f",
-            clientSecret: "e97051221f4b6426e8fe8d51486396703012f5bd"
+          ThirdParty.Github({
```

Comment:
you cannot use thirdparty.google with this. You need to use thirdpartyemailpassword.google.

Please undo all changes to the nodejs code snippets in this PR.

Reply:
Fixed that issue.